src/generator/models.ts

```diff
@@ -62,6 +66,17 @@ export async function generateModels(session: Session<CodeModel>): Promise<strin
     text += '\t}\n';
     text += '}\n\n';
   }
+  if (needsJSONPopulateByteArray) {
+    text += 'func populateByteArray(m map[string]interface{}, k string, b []byte, f azcore.Base64Encoding) {\n';
+    text += '\tif len(b) == 0 {\n';
+    text += '\t\treturn\n';
+    text += '\t} else if azcore.IsNullValue(b) {\n';
+    text += '\t\tm[k] = nil\n';
+    text += '\t} else if !reflect.ValueOf(b).IsNil() {\n';
```

Comment:
Just checking do we need to do the IsNullValue and IsNil with those azcore methods for a byte array just like we did for the interfaces?

Reply:
D'oh I think you're right, this can be simplified.  I also need to test the JSON `null` scenario too.

Reply:
Fixed with test.  We do need the `IsNullValue()` to handle the JSON null scenario.